core/src/main/java/org/bitcoinj/core/AbstractBlockChain.java

```diff
@@ -106,7 +106,7 @@
 
     // Holds a block header and, optionally, a list of tx hashes or block's transactions
     class OrphanBlock {
-        final Block block;
+        final AbstractBlock block;
```

Comment:
Why is this now an `AbstractBlock`? The constructor still can take only `Block`s.